### ISSUE TITLE
Replace common symbols for string alternative

### ISSUE THREAD

**Issue by Akxe** [prisma/prisma#2362]
I am migrating an existing database to Prisma V2 and I encountered a problem with enums.

## Problem

MySQL enum is as follow: `ENUM('Kč', '€', 'c€')`

This gets renamed to:
```
enum zakazky_zMena {
  K_ @map("Kč")
  // € @map("€")
  c_ @map("c€")
}
```

This apply to example bellow too:

```
enum svoobdobi_svoStav {
  bal__ek @map("balíček")
  pron_jem @map("pronájem")
  rezervace
  ciz_ @map("cizí")
  voln_ @map("volná")
}
```

### Possibly easy solution
Accented symbols could be transpired to their non-accented versions. (`á => a`, `é => e`, `ě => e`, etc...)
Common symbols like `$` could be translated into `dollar`.

### Better solution
Have some file that would serve as hash map for these symbols.

```
á => a
ě => e
é => e
€ => eur
$ => dollar
& => _and_
```

## Alternatives
Renaming it after every database iteration. Sounds painful right?

## Additional context
Prisma V2 beta.4

**Comment by Akxe** [comment]
I think this could be added to #2829 too.

**Comment by janpio** [comment]
No, this is in the right place @Akxe, and has nothing to do with Re-Introspection but a new feature Introspection Configuration. When we get to that, we will take this into consideration.

**Comment by albertoperdomo** [comment]
>Renaming it after every database iteration. Sounds painful right?

This should be no longer the case now with re-introspection. My understanding is that the existing mappings would be honored. Given this, the request is still legitimate, but the pain of not having the solution has somehow decreased.

**Comment by gabrielew** [comment]
I created a migration to handle with this issue

```js
-- DropForeignKey
DROP EXTENSION IF EXISTS unaccent;

-- AddForeignKey
CREATE EXTENSION unaccent;
```

Unfortunately, this only works with **query raw**!
